### PR TITLE
SAR-9908 Implement Hmrc-mongo

### PR DESCRIPTION
**Pull request by woollett-r**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ object AppDependencies {
 
   val compile = Seq(
     "uk.gov.hmrc"                  %% "bootstrap-backend-play-28"  % "5.3.0",
-    "uk.gov.hmrc"                  %% "simple-reactivemongo"       % "8.0.0-play-28",
+    "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-28"         % "0.66.0",
```
Comment (EviNikolaidou) [inline project/AppDependencies.scala]: the latest version is 0.67.0

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -20,7 +20,7 @@ object AppDependencies {
     "org.scalatestplus.play"  %% "scalatestplus-play"         % "5.1.0"             % "test, it",
     "com.github.tomakehurst"  %  "wiremock-jre8"              % "2.28.0"            % IntegrationTest,
     "org.mockito"             %% "mockito-scala"              % "1.16.37"           % Test,
-    "org.mockito"             %% "mockito-scala-scalatest"    % "1.16.37"           % Test
-
+    "org.mockito"             %% "mockito-scala-scalatest"    % "1.16.37"           % Test,
+    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-test-play-28"    % "0.66.0"            % Test
```
Comment (EviNikolaidou) [inline project/AppDependencies.scala]: the latest version is 0.67.0

File: project/AppDependencies.scala (hunk 1 above)
Comment (woollett-r) [reply project/AppDependencies.scala]: Done

File: project/AppDependencies.scala (hunk 2 above)
Comment (woollett-r) [reply project/AppDependencies.scala]: Done

**Review by EviNikolaidou — APPROVED** [review]
LGTM

**Pull request merged**